src/components/EntityPage/Participant/ParticipantClinical.js

```diff
@@ -133,6 +136,12 @@ class ParticipantClinical extends React.Component {
     this.phenotypeIntoState(api);
   }
 
+  prettySharedWith(amount) {
+    if(amount === "--") return amount;
+    else if(amount === 0 || amount === 1) return `${amount} participant`;
```

Comment:
In english, we have to write `0 participants`

https://english.stackexchange.com/a/13075/89065